src/main/resources/jenkins/plugins/openstack/compute/auth/OpenstackCredentialv2/help-tenant.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    The tenant associated to the user to start the machine.
```

Comment:
I find the "to start the machine" part in auth helps a bit confusing - it is needed for any interaction with openstack. I guess we can leave it out.